docs/DeveloperGuide.md

```diff
@@ -237,70 +237,403 @@ _{Explain here how the data archiving feature will be implemented}_
+| **Communication** |
+| `* * *`  | Product Manager                          | distinguish between the developers and the stakeholders in a project easily | remember the different roles       |
+| `*`      | Product Manager                          | view contacts of people involved in a product                               | communicate with relevant people                          |
+| `*`      | Product Manager                          | check the product’s completion stage                                        | quickly communicate any needs to the Dev Team and/or stakeholders |
+
+
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
 
-*{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Productiv` and the **Actor** is the `user`, unless specified otherwise)
+
+#### Mode
+
+**Use case: `UC01 - Switch Mode`**
+
+**MSS**
+
+1. User chooses to switch the mode.
+2. User enters the command to switch mode into the input box.
+3. Productiv switches to the expected mode.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. Produtiv detects an error in the command.
+     
+     * 2a1. Productiv displays an error message
+     
+     * 2a2. User enters the command again
+     
+     * 2a3. Steps 2a1-2a2 are repeated until the command entered is correct.
+     
+       Use case resumes from step 3.
+
+
+#### CONTACT
+**Use case: `UC10 - Add a contact`**
+
+**Precondition(s):**
+* **`User is in the Contact mode`**
+
+**MSS**
+
+1. User adds contact
```

Comment:
Noticed the use cases in this file is not reflected in our gdoc. Would be best to sync up both documents. But not the greatest priority now. We'll be basing on this markdown in the future I suppose. Just a note.